gnomad_constraint/resources/resource_utils.py

```diff
@@ -79,6 +89,32 @@ def get_preprocessed_ht(
     return TableResource(preprocessed_ht_path)
 
 
+def get_training_dataset(
+    version: str = CURRENT_VERSION,
+    genomic_region: str = "autosome_par",
+    test: bool = False,
+) -> TableResource:
+    """
+    Return TableResource of training dataset with observed and possible variant count.
+
+    :param version: One of the release versions (`VERSIONS`). Defaults to `CURRENT_VERSION`.
+    :param genomic_region: The genomic region of the resource. One of "autosome_par", "chrx_non_par", or "chry_non_par". Defaults to "autosome_par".
+    :param test: Whether the Table is for testing purpose and only contains sites in chr20, chrX, and chrY. Defaults to False.
+    :return: TableResource of training dataset.
+    """
+    if genomic_region not in GENOMIC_REGIONS:
+        raise ValueError(f"genomic_region must be one of: {GENOMIC_REGIONS}!")
+    if version not in VERSIONS:
+        raise ValueError("The requested version doesn't exist!")
+    training_dataset_path = f"{constraint_tmp_prefix}/model/training/constraint_training.{genomic_region}{'.test' if test else ''}.ht"
+    if not file_exists(training_dataset_path):
+        logger.info(
```

Comment:
It's the best way I can think about to alert user they shouldn't read a table that is not exist.